lib/truffle/java/java.rb

```diff
@@ -147,9 +147,6 @@ def method_missing(name, *args)
       val = JavaUtilities.get_relative_package_or_class(self, name)
       @children[name] = val
 
-      if name[0] == name[0].upcase && !val.kind_of?(JavaPackage)
-        const_set(name, val)
-      end
```

Comment:
Is this change intentional?

Reply:
Yes. It turns out constants was being set in two places when resolving a Java class so each lookup produced a warning.